andriy.safronov/01.rb

```diff
@@ -0,0 +1,10 @@
+a=0
```

Comment:
to avoid having 2 nested cycles, you can use method #flatten to make array one dimentional